### PR TITLE
add topic description for whats new files

### PR DESCRIPTION
**Pull request by BillWagner**
what's new files are overviews of what's new.

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -191,7 +191,8 @@
         "docs/framework/additional-apis/**/**.md": "managed-reference",
         "docs/framework/unmanaged-api/**/**.md": "reference",
         "docs/fsharp/language-reference/**/**.md": "language-reference",
-        "docs/standard/**/*how-to*.md": "how-to"
+        "docs/standard/**/*how-to*.md": "how-to",
+        "docs/whats-new/*.md" : "overview"
```
Comment (BillWagner) [inline docfx.json:195]: I'm not sure this is the best choice. If you have suggestions @tdykstra let me know.

Comment (tdykstra) [reply docfx.json:195]: my suggestion is "landing-page" to get the most appropriate KPI Score formula -- I've sent a query asking if we can use that value on a page that isn't strictly speaking a landing page.

Comment (tdykstra) [reply docfx.json:206]: Kelly is still looking into creating a new type -- until that happens, what's new docs will have to remain as conceptual.

Comment (tdykstra) [reply docfx.json:206]: It doesn't look like creating a new type is going to happen anytime soon -- should we close this PR?

**Review by IEvangelist — APPROVED** [review]
I think that `overview` is more appropriate - at least for now. Looks good to me, :shipit: when you're ready.

**Comment by IEvangelist** [comment]
Is this still needed @BillWagner?

**Comment by BillWagner** [comment]
closing as we haven't decided which topic type to use.

**Pull request closed without merging**